core/src/main/resources/hudson/model/Job/buildTimeTrend_resources.js

```diff
@@ -122,3 +122,82 @@ function generateSVGIcon(iconName, iconSizeClass) {
 
   return span;
 }
+
+/**
+ * Public method to be called by progressiveRendering's callback
+ */
+window.displayBuilds = function (data) {
+  var rootUrl = document.head.getAttribute("data-rooturl");
+  var p = document.getElementById("projectStatus");
+  p.style.display = "";
+  for (var x = 0; data.length > x; x++) {
+    var e = data[x];
+    var tr = document.createElement("tr");
+
+    var td1 = document.createElement("td");
+    td1.setAttribute("data", e.iconColorOrdinal);
+    td1.classList.add("jenkins-table__cell--tight", "jenkins-table__icon");
+    var div1 = document.createElement("div");
+    div1.classList.add("jenkins-table__cell__button-wrapper");
+    var svg = generateSVGIcon(e.iconName, p.dataset.iconSizeClass);
+    div1.appendChild(svg);
+    td1.appendChild(div1);
+    tr.appendChild(td1);
+
+    var td2 = document.createElement("td");
+    var a1 = document.createElement("a");
+    a1.classList.add("jenkins-table__link", "model-link");
+    a1.href = rootUrl + "/" + e.parentUrl;
+    var span1 = document.createElement("span");
+    span1.textContent = e.parentFullDisplayName;
+    a1.appendChild(span1);
+    td2.appendChild(a1);
+    var a2 = document.createElement("a");
+    a2.classList.add(
+      "jenkins-table__link",
+      "jenkins-table__badge",
+      "model-link",
+      "inside"
+    );
+    a2.href = rootUrl + "/" + e.url;
+    a2.textContent = e.displayName;
+    td2.appendChild(a2);
+    tr.appendChild(td2);
+
+    var td3 = document.createElement("td");
+    td3.setAttribute("data", e.timestampString2);
+    var button = document.createElement("button");
+    button.classList.add("jenkins-table__link");
+    button.setAttribute("tooltip", p.dataset.scrollTooltip);
+    button.onclick =
+      'javascript:tl.getBand(0).scrollToCenter(Timeline.DateTime.parseGregorianDateTime("' +
+      e.timestampString3 +
+      '"))';
```

Comment:
This doesn't work, the resulting button has no `onclick` attribute, so no scrolling takes place when clicking the "Time Since" value.

Reply:
commit 00a1245307